### PR TITLE
Midround alien larvas start the game already in a vent

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

ports https://github.com/tgstation/tgstation/pull/59567

# Wiki Documentation

# Changelog

:cl: Arkatos1, ported by TheGamerdk
rscadd: Midround alien larvas start the game already in a vent instead of just on top of it. Fair chance for everyone!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/ventcrawling.dm [hunk 1]
```diff
@@ -68,7 +68,7 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, typecacheof(list(
 					return
 
 			visible_message(span_notice("[src] scrambles into the ventilation ducts!"),span_notice("You climb into the ventilation ducts."))
-			forceMove(vent_found)
+			move_into_vent(ventcrawl_target)
```
Comment (Bibby0110) [inline code/modules/mob/living/ventcrawling.dm]:
```suggestion
			move_into_vent(vent_found)
```

File: code/modules/mob/living/ventcrawling.dm [hunk 2]
```diff
@@ -78,6 +78,18 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, typecacheof(list(
...
 
+/**
+ * Moves living mob directly into the vent as a ventcrawler
+ *
+ * Arguments:
+ * * ventcrawl_target - The vent into which we are moving the mob
+ */
+/mob/living/proc/move_into_vent(obj/machinery/atmospherics/components/ventcrawl_target)
+	forceMove(ventcrawl_target)
+	ADD_TRAIT(src, TRAIT_MOVE_VENTCRAWLING, VENTCRAWLING_TRAIT)
```
Comment (tattax) [inline code/modules/mob/living/ventcrawling.dm]:
```suggestion

```

Comment (Bibby0110) [inline code/modules/mob/living/ventcrawling.dm]:
```suggestion
```

**Comment by MusselMain** [comment]
Gud change because I've died immediately as an alien larva at least twice because funni larva spawn

**Comment by ghost** [comment]
Sus

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**